en/components/rating.md

```diff
@@ -0,0 +1,89 @@
+---
+
+# Rating
+
+The Rating component enables users to apply a rating to an item or experience, such as a product, service, page, other entity, and more. The Rating can display a single user's, or an aggregate, star rating and it's used to let users set their own star rating.
+
+## Rating Demo
+
+<img class="responsive-img" src="../images/rating_demo.png" srcset="../images/rating_demo@2x.png 2x" />
+
+## Using the Rating in Figma
+
+The Rating is built from Label and Value, where the Value is the number of the rating scale. The number of icons in the rating scale is predefined up to five but if you need to design a Rating with more rating icons, the easiest way to do so is by inserting the component into your workspace. After that, right-clicking on the component will open a contextual menu where `Detach Instance` should be selected, and you should see the following detached layer structure:
+
+| Layer                        | Use                                                                                                                                                  |
+| ---------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 🚫 componentVersion | A special locked layer starting with a prohibited icon. This layer is required by the code generation and you should avoid deleting or modifying it. |
+| 🚫 metadata         | A special locked layer starting with a prohibited icon. This layer is required by the code generation and you should avoid deleting or modifying it. |
+| Label              |                                                                                                                 |
+| Value              | Contains all the rating icons                                                                                                                 |
+
+After detaching, you may increase the number of the rating icons by duplicating any of the existing ones nested in the `Value` frame, and the size will be adjusted thanks to the Auto Layout.
+
+To decrease the number of the rating icons, there is no need to detach the component, you should hide the not needed one from the Layers panel.
+
+## Using the Rating in Sketch
+
+The Raiting component in Sketch has almost the same building elements like in Figma. There are Label and Value and the component comes with five rating icons by default. In case that there is need for more rating icons the component needs to be detached by right-clicking on the component and selecting `Detach from Symbol` from the contextual menu. After detaching you should see the following detached layers:
```

Comment:
In case that there is need for more rating icons **,** the component needs...

Reply:
Fixed in the next commit